tests/execs.rs

```diff
@@ -0,0 +1,60 @@
+use std::{env, fs, path::PathBuf};
+
+use duct::cmd;
+use nix::sys::stat::{stat, Mode};
+
+const DIR: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/tests");
+
+const CLIENT_BIN: &str = env!("CARGO_BIN_EXE_client");
+const SERVER_BIN: &str = env!("CARGO_BIN_EXE_server");
+
+#[test]
+fn run_execs() {
```

Comment:
Looks quite heavy to run the tests from rust - what is the advantage over running it from a shell?

Reply:
Discussed face2face: integrations tests are expected to be run by `cargo test`